gtda/images/preprocessing.py

```diff
@@ -203,9 +214,20 @@ class Inverter(BaseEstimator, TransformerMixin, PlotterMixin):
 
     """
 
-    def __init__(self, n_jobs=None):
+    _hyperparameters = {
+        'max_value': {'type': (Real, type(None))}
```

Comment:
Shouldn't we add `bool` (or `np.bool`, which is just an alias) to the tuple of allowed types?